pfcpiface/parse-far.go

```diff
@@ -18,10 +18,12 @@ const (
 )
 
 type far struct {
-	farID uint32
-	fseID uint32
+	farID   uint32
+	fseID   uint32
+	fseidIP uint32
 
 	action       uint8
+	applyAction  uint8
```

Comment:
merge these two

Reply:
the action params used in bess is different than what comes in the flags. p4rt is using the flags directly as per spec like (NotifyCP, buffer, Forward) if I keep the variable common It will impact the bess flow. so keeping it separate for now.

Reply:
keep this layer as per spec and rectify bess implementation.

Reply:
https://github.com/omec-project/upf-epc/issues/192 : will fix this in a separate ticket.